reader/src/class_file_method.rs

```diff
@@ -104,7 +104,7 @@ impl fmt::Display for ClassFileMethodCode {
                 writeln!(f, "    {address:3} {instruction:?}")?;
             }
         } else {
-            writeln!(f, "    unparseable code: {:?}", self.code)?;
+            writeln!(f, "    unparsable code: {:?}", self.code)?;
```

Comment:
https://en.wiktionary.org/wiki/unparseable

Reply:
I've scrolled through all of the 78 google results for https://www.google.com/search?q=%22unparseable%22+%22dictionary%22 , and there is only wiktionary and copies of wiktionary.  Not one authoritative source.